src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -330,4 +359,40 @@ private void printType(PrintWriter out, GraphQLType type) {
         TypePrinter<Object> printer = printer(type.getClass());
         printer.print(out, type);
     }
+
+
+    private void printComments(PrintWriter out, Object graphQLType, String prefix) {
+        String description = getDescription(graphQLType);
+        if (description == null) {
+            return;
+        }
+        Stream<String> stream = Arrays.stream(description.split("\n"));
+        stream.map(s -> prefix + "#" + s + "\n").forEach(out::write);
+    }
+
+    private String getDescription(Object descriptionHolder) {
```

Comment:
Yes ... there is no common type here and I didn't want to introduce one just to make it more "nice" here